sentry/src/main/java/io/sentry/ConvertibleToEnvelopeItem.java

```diff
@@ -0,0 +1,6 @@
+package io.sentry;
+
+/** An object that can be converted to a {@link SentryItem} as a part of {@link SentryEnvelope}. */
+public interface ConvertibleToEnvelopeItem {
```

Comment:
I am not happy with this name and am very open for suggestions.

Reply:
IEnvelopeItemConverter maybe?

Reply:
It's not a converter though, just something that can be converted.

Reply:
`SentryEventMarker` or `SentryEventBaseMarker` then if its a marker interface pattern